schemas/pci/pci-bus.yaml

```diff
@@ -114,6 +114,9 @@ properties:
   supports-clkreq:
     type: boolean
 
+  aspm-no-l0s:
```

Comment:
Please include the description you had.

Reply:
Hi Rob, do you mean within the commit message or as a comment in pci-bus.yaml?  I did not see any other comments in the latter.  -- Jim

Reply:
under a 'description' key. The others don't yet because I need relicense the text from the kernel pci.txt.

Reply:
I updated branch "patch-1" with the aspm-no-l0s description; is that enough or should I initiate another pullreq?  Thanks, Jim